packages/cli/__tests__/zosfiles/__integration__/create/ds/__scripts__/command/command_create_ds_fail_missing_like.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+echo "================Z/OS FILES CREATE DS==============="
+zowe files create data-set my.data.copy 
```

Comment:
Please add host, user, and password arguments to this command, so that it skips prompting for those values and fails due to missing like data set name as desired. For example:
`zowe files create data-set my.data.copy --host fakehost --user fakeuser --pw fakepass`